brave/browser-laptop
# Add "wide URL bar" option

Fix #8421

Auditors:

Test Plan 1:
1. Clear the browser profile
2. Open the browser
3. Open about:preferences#general
4. Make sure "Use wide URL bar" is disabled
5. Enable the option
6. Make sure the URL bar fills the margin around itself

Test Plan 2:
1. Open about:preferences#extensions
2. Enable LastPass extension
3. Make sure there is margin between the URL bar and the extension's icon

Test Plan 3:
1. Minimize the window's width
2. Make sure there does not appear extraDragArea between the URL bar and the extension's icon


Test Plan:
